nibe/console_scripts/convert_csv.py

```diff
@@ -158,17 +170,25 @@ def calculate_number(register_type: str, register: str):
             return None
 
         if "id" not in self.data:
-            self.data["id"] = self.data["registertype"].combine(
-                self.data["register"], calculate_number
+            id_prefixed = (self.data["title"].str.startswith("id:")) | (
+                self.data["title"] == "-"
```

Comment:
This is rather ugly. There are duplicated registers in some files. These seem to be unnamed and just have a "-" as title or a title like "id:".

I want back and forth, but landed on just ignoring these unnamed registers. If we want them they should be adjusted.